### DISCUSSION TITLE
suggestion to change name

### DISCUSSION THREAD

**Discussion by theArianit** [oven-sh/bun#1681] [General]
Hello. I hope I don't offend anyone with my opinion. But can the name of bun be changed before stable release? There are so many cool names to come up with. But from all those existing names and others to make up, from all the spoken human languages, bun seems to be a name that can't be taken serious from companies. Just imagine you go to your boss and try to suggest him to switch from node to something else and you bring up the name bun.
What could make the leaders of this project consider this suggestion?

**Comment by Jarred-Sumner** [comment]
I definitely appreciate the feedback, but I think bun is a good name. It's a **bun**dling of the JavaScript ecosystem, a **bun**dler and the terminal command is short.

The bigger issue with respect to bun being taken seriously right now is the lack of documentation, lack of logos, beta state, and things like that

**Reply by theArianit** [reply]
thanks for your reply, appreciate it. and I see what you did there, smart :)

**Reply by chebread** [reply]
@Jarred-Sumner, I also think Bun is a good name!
